docker-compose.yml

```diff
@@ -7,9 +7,6 @@ services:
     ports:
       - 3000:3000
       - 35279:35729
-    command: npm start
-    volumes:
-      - ./client:/app
```

Comment:
Are we keeping or dropping volumes?

Reply:
I'll reintroduce them in a following pull request, maybe later today.